helpers/Authoring.php

```diff
@@ -123,5 +112,48 @@ public static function addRequiredResources($sourceDirectory, $relativeSourceFil
 
         return $returnValue;
     }
-
+    
+    /**
+     * Remove invalid elements and attributes from QTI XML. 
+     * @param string $qti
+     * @return string sanitized XML
+     */
+    public static function sanitizeQtiXml($qti)
+    {
+        $doc = self::loadQtiXml($qti);
+        
+        $xpath = new \DOMXpath($doc);
+        
+        foreach ($xpath->query("//*[local-name() = 'itemBody']//*[@style]") as $elementWithStyle) {
+            $elementWithStyle->removeAttribute('style');
+        }
+        
+        return $doc->saveXML();
+    }
+    
+    /**
+     * Load QTI xml and return DOMDocument instance. 
+     * If string is not valid xml then QtiModelException will be thrown.
+     * @param string $qti
+     * @throws QtiModelException
+     * @return DOMDocument
+     */
+    public static function loadQtiXml($qti) 
+    {
+        $errors = array();
+        
+        $dom = new DOMDocument('1.0', 'UTF-8');
+        $dom->formatOutput = true;
+        $dom->preserveWhiteSpace = false;
+        $dom->validateOnParse = false;
+        
+        libxml_use_internal_errors(true);
+        
+        if (!$dom->loadXML($qti)) {
+            $errors = libxml_get_errors();
+            throw new QtiModelException('Wrong QTI item output format');
```

Comment:
Should not we use the $errors to display a more explicit error message in the QtiModelException